libs/theme/src/offerMasses/offerMass.interface.ts

```diff
@@ -0,0 +1,30 @@
+import { Dayjs } from 'dayjs';
+
+export interface OfferMassContextProviderProps {
+  children: JSX.Element;
+}
+
+interface Requester {
```

Comment:
restructure requester's data here to make it either a person nor anonymous not having both at the same type.